src/apps/chifra/pkg/rpcClient/get_transaction.go

```diff
@@ -85,7 +85,7 @@ func GetTransactionByAppearance(chain string, appearance *types.RawAppearance, f
 		Txid:    txid,
 		NeedsTs: true,
 		Ts:      blockTs,
-	})
+	}, nil)
```

Comment:
Unsearchable

Reply:
This is different. In this case we don't want to look up Receipt in the cache, because we know it's not there. This code is executed only if the transaction has not been cached.